components/TopBarProfileMenu.js

```diff
@@ -128,11 +148,11 @@ class TopBarProfileMenu extends React.Component {
         {isAdmin && (
           <Link route="editCollective" params={{ slug: membership.collective.slug }} passHref>
             <StyledLink color="black.500" title={intl.formatMessage(this.messages.settings)}>
-              <Settings size="1.2em" />
+              <Settings className="SettingsCogIcon" opacity="0" size="1.2em" />
```

Comment:
As per the previous comment, we should remove `className`. I'm also surprised by this `opacity` prop, does it serves a particular purpose?

Reply:
I've removed the `className` as you suggested. The opacity needs to be there I think since otherwise for desktop it will always show the cog icon. Notice that at the top we set opacity to 1. Here by default it's 0. :smile: